docs/src/examples/components/List/Types/ListExampleActive.shorthand.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { List, Image } from '@stardust-ui/react'
+
+class ListExampleActive extends React.Component<any, any> {
```

Comment:
No TS in example files for now, please.  The majority of users are not using TS.  We plan to address a better way of handling this in the future.

```suggestion
class ListExampleActive extends React.Component {
```

Reply:
It is giving the following error without it. How to resolve this?
![screenshot 28](https://user-images.githubusercontent.com/11490517/47314214-ac40d500-d65e-11e8-9a32-ff279f4eec28.png)

Reply:
@levithomason @kuzhelov